subprojects/base-services/src/main/java/org/gradle/internal/agents/AgentControl.java

```diff
@@ -45,22 +48,56 @@ public static boolean isInstrumentationAgentApplied() {
     }
 
     private static boolean isAgentApplied(String agentClassName) {
+        Method isAppliedMethod = findAgentMethod(INSTRUMENTATION_AGENT_CLASS_NAME, "isApplied");
```

Comment:
💄 I guess the parameter could be removed now?

Reply:
Probably we don't need the separate method either